benchmark/streamly-benchmarks.cabal

```diff
@@ -226,6 +225,18 @@ benchmark serial-o-n-space
     build-depends: lib-prelude
     buildable: True
 
+-- XXX Default memory restriction = o-1-space
+benchmark fold
+  import: bench-options
+  type: exitcode-stdio-1.0
+  ghc-options: -with-rtsopts "-T -K36K -M16M"
```

Comment:
is 16M heap enough for o-n-heap fold benchmarks?

Reply:
I think we should remove ghc-options all togather. We'll control that via `bench.sh`.